src/supernova/test.rs

```diff
@@ -88,55 +38,58 @@ where
     }
   }
 }
-/// c = a + b where a, b is `AllocatedNum`
-pub fn add_allocated_num<F: PrimeField, CS: ConstraintSystem<F>>(
```

Comment:
In case this comes up one day, https://github.com/lurk-lab/bellpepper/blob/main/crates/bellpepper-core/src/gadgets/num.rs#L279-L309